modules/user_accounts/php/edit_user.class.inc

```diff
@@ -202,7 +202,11 @@ class Edit_User extends \NDB_Form
         $permIDs = array();
         // store the permission IDs
         if (!empty($values['permID'])) {
-            $permIDs = array_keys($values['permID']);
+            // An editor can only grant permission that he is granted of.
```

Comment:
I would change this to "An editor can only grant permissions that they have already been granted"